.gitignore

```diff
@@ -41,4 +41,4 @@ dist
 .vscode
 
 # generated code
-lib
\ No newline at end of file
+# lib
```

Comment:
If this isn't needed, then we should remove it instead of commenting it out

Reply:
I talked with @RyanCommits and this will be reverted after testing is finished.

Reply:
Opened a new branch to include the lib. Removed the change in this one